pages_/shop-form.tsx

```diff
@@ -495,6 +496,17 @@ const EditShop = ({ isLoggedIn, shop }: Props) => {
             >
               {t('common:save')}
             </LoadingButton>
+            <Button
+              block
+              href="/"
+              type="button"
+              disabled={isSubmitting}
+              variant="danger"
+              size="lg"
+              className="mt-4"
+            >
+              {t('common:cancel')}
+            </Button>
```

Comment:
This should be wrapped with a Link as other parts of the app. It should also link to /my-shop

Reply:
This is a form, I think having a Cancel button is the best approach. But we can have both, to be consistent with other screens.

Reply:
I mean, the button is ok, but it needs to be wrapped with a Link so it enables client side routing

Reply:
done